src/wrappers/PhylorefWrapper.js

```diff
@@ -296,10 +296,22 @@ class PhylorefWrapper {
    *   for. We mainly use this to retrieve its '@id'.
    * - internalSpecifiers: The set of internal specifiers for this additional class.
    * - externalSpecifiers: The set of external specifiers for this additional class.
-   * - equivClassFunc: The equivalent class expression for this additional class as a function
+   * - equivClass: The equivalent class expression for this additional class as a function
    *   that returns the expression as a string.
+   * - reusePrevious (default: true): If true, we reuse previous expressions with the
+   *   same set of included and excluded specifiers. If false, we always generate a new
+   *   additional class for this expression.
+   * - parentClass: If not undefined, provides a JSON-LD definition of the class to set as the
+   *   parent class of this additional class. We only use the ['@id'].
    */
-  static createAdditionalClass(jsonld, internalSpecifiers, externalSpecifiers, equivClass) {
+  static createAdditionalClass(
```

Comment:
I'm really not a big fan of this name. "additional class" can mean so many things. How about something like `createExpressionComponentClass()`, or `createComponentClass()`?

Reply:
"Component class" works for me. I've renamed this in 607405f7b.